doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -33,830 +35,44 @@ The following sections provide detailed lists of changes by component.
 Application development
 =======================
 
-* Fixed an issue with |NCS| Toolchain delimiter handling on MacOS, which could in special situations result in the build system not being able to properly locate the correct program needed.
+|no_changes_yet_note|
 
 Protocols
 =========
 
 This section provides detailed lists of changes by :ref:`protocol <protocols>`.
 See `Samples`_ for lists of changes for the protocol-related samples.
 
-Bluetooth mesh
---------------
-
-* Added :c:struct:`bt_mesh_sensor_srv` context to relevant callbacks and APIs to help resolve the associated sensor model instance.
-  For details, see `Bluetooth mesh samples`_ and `Bluetooth libraries and services`_.
-
-See `Bluetooth mesh samples`_ for the list of changes for the Bluetooth mesh samples.
-
-Matter
-------
-
-* The CHIP Tool controller is now the recommended controller for Matter.
-  The documentation about :ref:`ug_matter_configuring_controller` has been updated accordingly.
-  For more information about the CHIP Tool controller, read the :doc:`matter:chip_tool_guide` page in the Matter documentation.
-* Added the :ref:`matter_window_covering_sample` sample, based on the Matter upstream sample.
-
-See `Matter samples`_ for the list of changes for the Matter samples.
-
-Matter fork
-+++++++++++
-
-The Matter fork in the |NCS| (``sdk-connectedhomeip``) contains all commits from the upstream Matter repository up to, and including, ``25e241ebcbf11b1f63dbe25546b1f10219866ad0``.
-
-The following list summarizes the most important changes inherited from the upstream Matter:
```

Comment:
I do not think we should be removing these (Matter fork details) ..I suggest retaining these sentences followed by a |no_changes_yet_note|.

Reply:
Done, and I also decided to retain all the other lower level sections in the changelog